addons/jest.js

```diff
@@ -33,8 +33,5 @@ module.exports = {
         }, {}),
     }, {
         'files': testFilePatterns,
-        'rules': {
-            'prefer-import/prefer-import-over-require': 0,
```

Comment:
Hmm I guess we want to keep this in the form of `'import/no-commonjs': 0`. The reason is that we have to use `require` when using [`jest.doMock`](https://jestjs.io/docs/en/jest-object#jestdomockmodulename-factory-options).